static/app/views/starfish/views/screens/screenLoadSpans/table.tsx

```diff
@@ -0,0 +1,187 @@
+import {Fragment} from 'react';
+
+import {getInterval} from 'sentry/components/charts/utils';
+import GridEditable, {GridColumnHeader} from 'sentry/components/gridEditable';
+import SortLink from 'sentry/components/gridEditable/sortLink';
+import Pagination from 'sentry/components/pagination';
+import {t} from 'sentry/locale';
+import {NewQuery} from 'sentry/types';
+import {TableDataRow} from 'sentry/utils/discover/discoverQuery';
+import EventView, {isFieldSortable, MetaType} from 'sentry/utils/discover/eventView';
+import {getFieldRenderer} from 'sentry/utils/discover/fieldRenderers';
+import {fieldAlignment, Sort} from 'sentry/utils/discover/fields';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {TableColumn} from 'sentry/views/discover/table/types';
+import {SpanFunction, SpanMetricsField} from 'sentry/views/starfish/types';
+import {STARFISH_CHART_INTERVAL_FIDELITY} from 'sentry/views/starfish/utils/constants';
+import {appendReleaseFilters} from 'sentry/views/starfish/utils/releaseComparison';
+import {QueryParameterNames} from 'sentry/views/starfish/views/queryParameters';
+import {useTableQuery} from 'sentry/views/starfish/views/screens/screensTable';
+import {DataTitles} from 'sentry/views/starfish/views/spans/types';
+import {useModuleSort} from 'sentry/views/starfish/views/spans/useModuleSort';
+
+const {SPAN_SELF_TIME, SPAN_DESCRIPTION, SPAN_GROUP, SPAN_OP, PROJECT_ID} =
+  SpanMetricsField;
+
+type Props = {
+  primaryRelease?: string;
+  secondaryRelease?: string;
+  transaction?: string;
+};
+
+export function ScreenLoadSpansTable({
+  transaction,
+  primaryRelease,
+  secondaryRelease,
+}: Props) {
+  const location = useLocation();
+  const {selection} = usePageFilters();
+  const organization = useOrganization();
+
+  const searchQuery = new MutableSearch([
+    'transaction.op:ui.load',
+    `transaction:${transaction}`,
```

Comment:
nit: also here theres a possibility that transaction is undefined so we might want to check the value before constructing the search query string

Reply:
Yeah, good point! I'll address these in a follow up!